makepackage.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+vendor/bin/box compile
```

Comment:
I don't like how this has the same name as "makepackage.sh" in elkarbackup directory, for building a .deb package. Maybe we can call the file "makephar.sh" or just "build.sh" ?